src/components/Layout/Layout.js

```diff
@@ -23,14 +27,19 @@ class Layout extends Component {
   render() {
     return (
       <React.Fragment>
-        <div className="App wrapper">
+        <div className="App">
           <SideBar toggle={this.toggle} isOpen={this.state.isOpen} />
-          <NavBarContainer
-            toggle={this.toggle}
-            isOpen={this.state.isOpen}
-            showDialog={this.handleShowDialog}
-          />
+          <Container
```

Comment:
@DerrickPersaud I deleted the NavBarContainer component. We need this Container component wrapped around the entire body of the app in order for the app to respond appropriately when toggling the sidebar on and off. This change does that